ansible/configs/ibm-rhoic/software.yml

```diff
@@ -52,88 +79,166 @@
         var: rhoic_openshift_version
       tags: get-versions
 
-    ## Cluster Creation
-    - name: Create RHOIC cluster
+- name: Building the Cloud Object Storage and RHOIC Cluster
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  tags:
+    - post_flight_check
+  tasks:
+    ## Authenticated from last section in this playbook
+    # Create Workspace to support RHOIC
+    - name: Create a new workspace
```

Comment:
I haven't looked at this in a while as you know, but can't we create multiple schematics in the same workspace rather than having different workspaces for each thing?

Reply:
From everything I can tell it is one terraform repository per workspace. IBM recommended two workspaces as well.
I also tried to only use one set of terraform scripts for creating the VPC and RHOIC cluster and it works for the build, but I had failures testing the destroy. The failure to destroy with the combined scripts would always be related to not being able to remove the VPC when it has network interfaces associated with it.